hatchways/team-bluejay
# Kareem signup login page (Client) #6

Ticket: #6 

Description: 
- Changed the default font to monsterrat
- Updated theme.js by changing the primary and secondary colors
- Added react routes for login and sign up
- added 2 new pages for Login and sign up
- Added 3 new components (Authpage, LoginForm, SignUpForm)
- Added a reusable form component in src/components/common/
- Added client side validation for sign up and login forms
- Added navigation between login and sign up pages
- Added 2 files to images folder, logo and Login/signup page image.
- Updated React router package to latest to add hook support
- Added react-hook-form package for managing form data and validation

Testing:
- Checkout branch and npm install  start
- Verify that you have been navigated to Sign up screen
- Verify that the form fields are correct
- test the client side validation (can be updated later if required)
    - Name : required, max 50 chars
    - Email: required and must be string@string.string
    - Password: required, At least 1 digit (0-9), At least 1 uppercase letter, At least one special character
    - Test if the confirm PW field works as expected
- Verify that the navigation between sign up and login works
- Verify that the design is responsive
- Verify that the theme colors have been updated to match the spec
- verify that the font matches the spec
- Review component structure and styling
